CONTRIBUTING.md

```diff
@@ -8,9 +8,9 @@ ask that you make note of the following guidelines.
 * **Follow the [Oracle coding conventions](http://www.oracle.com/technetwork/java/codeconv-138413.html).**
   We can't stress this enough; if your code has notable issues, it may delay
   the process significantly.
-* **Target Java 6 for source and compilation.** Make sure to mark methods with
+* **Target Java 8 for source and compilation.** Make sure to mark methods with
   ` @Override` that override methods of parent classes, or that implement
-  methods of interfaces (Java 6+).
+  methods of interfaces (Java 8+).
```

Comment:
I don't this one should've changed, although it makes sense to remove entirely probably